assets/js/blocks/active-filters/block-wrapper.tsx

```diff
@@ -11,12 +12,15 @@ import Block from './block';
 import { parseAttributes } from './utils';
 
 const BlockWrapper = ( props: Record< string, unknown > ) => {
-	const colorProps = useColorProps( props );
+	const styleProps = useStyleProps( props );
```

Comment:
Shouldn't this be `props.attributes` (there may be more occurrences of this)?

Reply:
It does seem so yes, good catch.

Reply:
I think I see the issue; its confusing because some blocks have wrappers. I can fix this.

Reply:
I found some blocks (e.g. attribute filters) which don't need frontend.ts and don't need style props to function. I'm removing those because the props passed were in the wrong format.

Reply:
@mikejolley are you still planning to update the name here? There are a few other instances where `props` is used too which may be worth renaming